x/market/handler/endblock_external_test.go

```diff
@@ -0,0 +1,131 @@
+package handler_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ovrclk/akash/app"
+	"github.com/ovrclk/akash/testutil"
+	"github.com/ovrclk/akash/testutil/state"
+	"github.com/ovrclk/akash/x/market/handler"
+	"github.com/ovrclk/akash/x/market/types"
+)
+
+func TestMatchOrders(t *testing.T) {
+	codec := app.MakeCodec()
+	suite := state.SetupTestSuite(t, codec)
+	t.Logf("block height: %d", suite.Context().BlockHeight())
```

Comment:
Remove debug logging before opening a PR.  I'm tired of having to point this out.